x-pack/plugins/security_solution/server/endpoint/services/actions/actions_audit_log.ts

```diff
@@ -32,13 +41,134 @@ const queryOptions = {
   ignore: [404],
 };
 
-export const getTimeSortedData = (data: ActivityLog['data']): ActivityLog['data'] => {
+/**
+ * Used only for the deprecated `/api/endpoint/action_log/{agent_id}` legacy API route
+ *
+ * Use newer response action services instead
+ *
+ * @deprecated
+ */
+export const getAuditLogResponse = async ({
```

Comment:
These were moved here since they are only supporting the old deprecated audit log api